src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Exavator" + " " + machineType + " " + "started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Exavator" + " " + machineType + " " + "stopped working.");

```

Comment:
The same typo is present in the 'stopWork' method. It should be corrected to 'Excavator'.